tests/headers/repr-align.hpp

```diff
@@ -9,3 +9,7 @@ struct alignas(double) b {
   int b;
   int c;
 };
+
+namespace std {
+union a;
```

Comment:
Oh, so it's a forward declaration, fun... Let me think a bit whether it's the right fix. Thank you so much for the test-case!